perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/SecurityTeamsManagerMethod.java

```diff
@@ -120,10 +121,50 @@ public SecurityTeam call(ApiCaller ac, Deserializer parms) throws PerunException
 	getAdmins {
 		@Override
 		public List<User> call(ApiCaller ac, Deserializer parms) throws PerunException {
-			return ac.getSecurityTeamsManager().getAdmins(ac.getSession(), ac.getSecurityTeamById(parms.readInt("securityTeam")));
-		}
+			if(parms.contains("onlyDirectAdmins")) {                       
+				return ac.getSecurityTeamsManager().getAdmins(ac.getSession(), 
+					ac.getSecurityTeamById(parms.readInt("securityTeam")),
+					 parms.readBoolean("onlyDirectAdmins"));
+			} else {
+				return ac.getSecurityTeamsManager().getAdmins(ac.getSession(),
+				ac.getSecurityTeamById(parms.readInt("securityTeam")),false);
+			}
+        	}             
 	},
 
+ /*#
+         * Get all SecurityTeam direct admins.
+         *
+         * @deprecated
+         * @param securityTeam int SecurityTeam  <code>id</code>
+         * @return List<User> list of admins of the SecurityTeam. 
+         */
+        getDirectAdmins {
+
+                @Override
+                public List<User> call(ApiCaller ac, Deserializer parms) throws PerunException {
```

Comment:
This one is also deprecated so remove it. We can use getAdmins with parameter "onlyDirectAdmins" instead.